modelon/impact/client/sal/workspace.py

```diff
@@ -66,10 +66,18 @@ def dependencies_get(
         return self._http_client.get_json(url)
 
     def workspace_export_setup(
-        self, workspace_id: str, publish: bool, class_path: Optional[str] = None
+        self,
+        workspace_id: str,
+        publish: bool,
+        class_path: Optional[str] = None,
+        access_settings: Optional[Dict[str, Any]] = None,
     ) -> Dict[str, Any]:
         url = (self._base_uri / "api/workspace-exports").resolve()
-        body = {"workspaceId": workspace_id, 'publish': publish}
+        body = {
+            "workspaceId": workspace_id,
+            "publish": publish,
+            "access": access_settings,
```

Comment:
I suggest we follow the pattern below for class-path and only add optional fields if they're not None. 
So:
```
if access_settings:
  body["access"] = access_settings
```

Reply:
The access_setting parameter would never be None(See https://github.com/modelon-community/impact-client-python/pull/255/files#diff-138c49731943a8bdb4b84b87ba6de4e5bc54993f304e31903e7462b3604a67cdR713). I felt that adding additional checks to see if groups names key in the access_setting would be not be worth to add in the client as we might add other keys as well in the future?

Reply:
Ok, you're right. I guess I find code like below a bit difficult to read:
```
  access_settings = {
            "groupNames": access.group_names
            if access and isinstance(access.group_names, list)
            else None
        }
```
I'd prefer
```
   if access:
        access_settings = {
            "groupNames": access.group_names # No need for "if hasattr(access, "group_names") else None"
        }
   else:
      access_settings = None
```
but I'm ok with your current implementation.

Reply:
I agree, this makes it clearer. I have made this change and pushed now! :)